hotnews/index.html

```diff
@@ -1,37 +1,107 @@
 <html>
-  <head>
-    <title>HotNews.ro</title>
-    <link rel="icon" type="image/png" sizes="32x32" href="./assets/favicon.png">
 
-    <script src="https://cdn.tailwindcss.com"></script>
-    <script>
-      tailwind.config = {
-        theme: {
-          extend: {
-            colors: {
-              hotblue: '#2145ac',
-              hotgreen: '#3f9863',
-              hotred: '#eb3223'
-            }
+<head>
+  <title>HotNews.ro</title>
+  <link rel="icon" type="image/png" sizes="32x32" href="./assets/favicon.png">
+  <link rel="stylesheet" href="hotnews.css">
+
+  <script src="https://cdn.tailwindcss.com"></script>
+  <script>
+    tailwind.config = {
+      theme: {
+        fontSize: {
+          font: '15px',
+          font2: '18px',
+          font3: '14px'
+        },
+        fontFamily: {
+          sans: ["Inter", "sans-serif"]
+        },
+        minheight: '90px',
+        extend: {
+          padding: {
+            paddingleft: '22px'
+          },
+          width: '1349px',
+          height: {
+            height: '90px',
+            height2: '52px',
+            height3: '22px',
+            height4: '50px'
+          },
+          colors: {
+            hotblue: '#2145ac',
+            hotgreen: '#3f9863',
+            hotred: '#eb3223',
+            hotblack: '#393939',
+            hotEconomie: '#ed5c43',
+            hotSport: '#36a562',
+            hotLife: '#d0459e',
+            hotScience: '#767fea',
+            hotRevista: '#484848',
+            hotDosare: '9d8562',
+            hotHome: '2145ac',
+            hotWhite: '#fff',
+            hotWar: '#ffa900'
           }
         }
       }
-    </script>
-  </head>
-  <body>
-    <div class="flex max-w-5xl mx-auto px-4 py-1.5 text-sm font-medium">
-      <div class="flex grow items-center">
-        <img src="https://hotnews.ro/static/assets/icons/ic_updated.svg" alt="updated_icon">
-        <div class="text-hotblue">Actualizat acum un minut</div>
-      </div>
-      <div class="flex shrink">
-        <div class="text-hotblue pr-1">EUR</div>
-        <div class="text-hotgreen pr-1">4.9395</div>
-        <div class="text-hotblue pr-1">USD</div>
-        <div class="text-hotred">4.8195</div>
-      </div>
+    }
+  </script>
+</head>
+
+<body>
+  <div class="flex max-w-5xl mx-auto px-4 py-1.5 text-sm font-medium text-[14px]">
```

Comment:
There is no need to define a custom text size rule (text-[14px]) because it's equivalent to text-sm which is already defined.